R/format_badges.R

```diff
@@ -0,0 +1,79 @@
+format_badge_url <- function(display, path) {
+  glue::glue("[{display}]({path})")
+}
+
+format_name_badge <- function(repo_path) {
```

Comment:
nit: the naming convention used is confusing to me. I kinda expected `format_badge_url()` to return a "formatted" badge URL (whatever that would be 🤣), but it returns markdown for a linked badge.

Reply:
I renamed it `build_markdown_link`